File: Python/Other algo/two_pointer_Ex.py
def isZero(arr):
    left = 0
    right = len(arr) - 1
    while(left < right):
        sum1 = arr[left] + arr[right]
        if sum1 == 0:
            return[arr[left], arr[right]]
        if sum1 < 0:
            left = left + 1
        else:
            right = right - 1
    return 0

print(isZero([-2,-1,0,2,4,5]))

File: Js/Other Algo/Two_pointer_ex.js
function isZero(arr){
  let left = 0;
  let right = arr.length - 1;
  while(left < right){
    sum = arr[left] + arr[right];
    if( sum == 0){
      return [arr[left], arr[right]];
    }
    if(sum > 0){
      right--;
    }else{
      left++;
    }
  }
}

console.log(isZero([-6,-3,-2,0,1,3,5,7]))